spec/factories/users.rb

```diff
@@ -112,14 +112,14 @@
     factory :dfe_user do
       is_support { true }
       email_address do
-        full_name.downcase.gsub(' ', '.') + ['@digital.education.gov.uk', '@education.gov.uk'].sample
+        full_name.downcase.gsub(' ', '.') + %w[@digital.education.gov.uk @education.gov.uk].sample
```

Comment:
Personally I think we need to get rid of factories using `sample` altogether: they've caused a lot of flakey specs

Reply:
Agreed. Not the goal of this PR but I will take the chance